src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'clear') {
+      for (const key in state) {
+        delete state[key];
+      }
+    }
+
+    if (actions[i].type === 'removeProperties') {
+      for (const key in state) {
+        if (actions[i].keysToRemove.includes(key)) {
```

Comment:
you don't need `if` here, if key doesn't exist in object delete will ignore this key